### PR TITLE
copy dygraph api to paddle.imperative

### PR DESCRIPTION
**Pull request by JepsonWong**
copy dygraph api from paddle.fluid.dygraph to paddle.imperative or paddle.

| paddle.fluid.dygraph | paddle |
| ---------- | ---------- |
| paddle.fluid.dygraph.enable_dygraph | paddle.enable_imperative |
| paddle.fluid.dygraph.disable_dygraph | paddle.disable_imperative |
| paddle.fluid.in_dygraph_mode | paddle.in_imperative_mode |
| paddle.fluid.dygraph.enabled | paddle.imperative.enabled |
| paddle.fluid.dygraph.grad | paddle.imperative.grad |
| paddle.fluid.dygraph.load | paddle.imperative.load |
| paddle.fluid.dygraph.save | paddle.imperative.save |
| paddle.fluid.dygraph.Layer | paddle.nn.Layer |

### CODE REVIEW

File: python/paddle/__init__.py [hunk 1]
```diff
@@ -227,3 +227,7 @@
 # from .framework import WeightNormParamAttr   #DEFINE_ALIAS
 # from .framework import Model   #DEFINE_ALIAS
 # from .framework import Sequential   #DEFINE_ALIAS
+from .fluid.dygraph.base import enable_dygraph  #DEFINE_ALIAS
+from .fluid.dygraph.base import disable_dygraph  #DEFINE_ALIAS
+enable = enable_dygraph  #DEFINE_ALIAS
```
Comment (phlrain) [inline python/paddle/__init__.py]: enable 改成 enable_imperative 
disable 改成 disable_imperative

File: python/paddle/imperative/__init__.py [hunk 2]
```diff
@@ -14,16 +14,17 @@
...
-    'BackwardStrategy', 'guard', 'Layer', 'LayerList', 'load_dygraph',
-    'save_dygraph', 'prepare_context', 'to_variable', 'TracedLayer', 'no_grad',
+    'BackwardStrategy', 'enabled', 'guard', 'Layer', 'LayerList', 'load',
+    'save', 'prepare_context', 'to_variable', 'TracedLayer', 'no_grad',
     'ParameterList', 'Sequential'
 ]
 
 from paddle.fluid import core
-from ..fluid.dygraph.base import guard, no_grad, to_variable
+from ..fluid.dygraph.base import enabled, guard, no_grad, to_variable
```
Comment (phlrain) [inline python/paddle/imperative/__init__.py]: 需要import 一下 grad, 二次梯度求导需要用的

**Comment by phlrain** [comment]
in_dygrah_mode  需要做一个alias in_imperative_mode

File: python/paddle/__init__.py (hunk 1 above)
Comment (JepsonWong) [reply python/paddle/__init__.py]: done

File: python/paddle/imperative/__init__.py (hunk 2 above)
Comment (JepsonWong) [reply python/paddle/imperative/__init__.py]: done

**Comment by JepsonWong** [comment]
> in_dygrah_mode 需要做一个alias in_imperative_mode

done

**Review by phlrain — APPROVED** [review]

**Review by Boyan-Liu — APPROVED** [review]

**Review by lanxianghit — APPROVED** [review]

**Pull request merged**